ext/WarMain.java

```diff
@@ -297,18 +282,6 @@ protected String locateExecutable(final Object scriptingContainer, final CharSeq
         }
     }
 
-    protected CharSequence executableScriptEnvPrefix() {
-        final String gemsDir = new File(extractRoot, "gems").getAbsolutePath();
-        final String gemfile = new File(extractRoot, "Gemfile").getAbsolutePath();
```

Comment:
This is the main issue. Our project is using a custom gemfile (with `BUNDLE_GEMFILE`) and it's getting overridden here